From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Mon, 8 Feb 2016 14:09:32 +0100
Subject: [PATCH] mtd: nand-bb: Fix 8k page size nands

As the comment states BB_WRITEBUF_SIZE must be a multiple of the largest
NAND page size. Since this is 8192 and not 4096, change it accordingly.

Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
---
 drivers/mtd/nand/nand-bb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/nand/nand-bb.c b/drivers/mtd/nand/nand-bb.c
index 8e4600ab03cd..5a2fd1096d9a 100644
--- a/drivers/mtd/nand/nand-bb.c
+++ b/drivers/mtd/nand/nand-bb.c
@@ -88,7 +88,7 @@ static ssize_t nand_bb_read(struct cdev *cdev, void *buf, size_t count,
 }
 
 /* Must be a multiple of the largest NAND page size */
-#define BB_WRITEBUF_SIZE	4096
+#define BB_WRITEBUF_SIZE	8192
 
 #ifdef CONFIG_MTD_WRITE
 static int nand_bb_write_buf(struct nand_bb *bb, size_t count)
